nodeup/pkg/model/logrotate.go

```diff
@@ -43,5 +46,70 @@ func (b *LogrotateBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	c.AddTask(&nodetasks.Package{Name: "logrotate"})
 
+	k8sVersion, err := util.ParseKubernetesVersion(b.Cluster.Spec.KubernetesVersion)
+	if err != nil || k8sVersion == nil {
+		return fmt.Errorf("unable to parse KubernetesVersion %q", b.Cluster.Spec.KubernetesVersion)
+	}
+
+	if k8sVersion.Major == 1 && k8sVersion.Minor < 6 {
+		// In version 1.6, we move log rotation to docker, but prior to that we need a logrotate rule
+		b.addLogRotate(c, "docker-containers", "/var/lib/docker/containers/*/*-json.log", logRotateOptions{MaxSize: "10M"})
+	}
+
+	b.addLogRotate(c, "docker", "/var/log/docker.log", logRotateOptions{})
```

Comment:
Do we want logrotate on `/var/log/messages` at all? Just curious (I think EBS can spam this hard)